alws/utils/pulp_client.py

```diff
@@ -308,6 +308,63 @@ async def sync_rpm_repo_from_remote(self, repo_href: str, remote_href: str,
             return result
         return task
 
+    async def create_filesystem_exporter(self, fse_name: str, fse_path: str,
+                                         fse_method: str='write'):
+        endpoint = 'pulp/api/v3/exporters/core/filesystem/'
+
+        params = {
+            'name': fse_name,
+            'path': fse_path,
+            'method': fse_method
+        }
+        result = await self.make_post_request(endpoint, params)
+        return result['pulp_href']
+
+    async def update_filesystem_exporter(self, fse_pulp_href: str,
+                                         fse_name: str,
+                                         fse_path: str,
+                                         fse_method: str='symlink'):
+        endpoint = fse_pulp_href
+        params = {
+            'name': fse_name,
+            'path': fse_path,
+            'method': fse_method
+        }
+        update_task = await self.make_put_request(endpoint, params)
+        task_result = await self.wait_for_task(update_task['task'])
+        return task_result
+
+    async def delete_filesystem_exporter(self, fse_pulp_href: str):
+        delete_task = await self.make_delete_request(fse_pulp_href)
+        task_result = await self.wait_for_task(delete_task['task'])
+        return task_result
+
+    async def list_filesystem_exporters(self):
+        endpoint = 'pulp/api/v3/exporters/core/filesystem/'
+        result = await self.make_get_request(endpoint)
+        if result['count'] > 0:
+            return result['results']
+        else:
+            return []
+
+    async def get_filesystem_exporter(self, fse_pulp_href : str):
+        endpoint = fse_pulp_href
+        result = await self.make_get_request(endpoint)
+        return result
+
+    async def export_to_filesystem(self, fse_pulp_href: str,
+                                   fse_repository_version: str):
+        endpoint = urllib.parse.urljoin(fse_pulp_href, 'exports/')
+        params = {
+            'repository_version': fse_repository_version
+        }
+        result = await self.make_post_request(endpoint, params)
```

Comment:
May be there is a point to add in this functions `wait_for_task` method and delete from `alws/crud/repo_exporter.py:99`?

Reply:
Fixed. Thanks!